python/tvm/topi/cuda/conv2d_nhwc_winograd.py

```diff
@@ -432,7 +441,8 @@ def nhwc_winograd_cuda(
         name="output",
         tag="conv2d_nhwc_winograd",
     )
-    cfg.add_flop(2 * N * CO * H * W * CI * KH * KW)
+    if isinstance(N, int):
+        cfg.add_flop(2 * N * CO * H * W * CI * KH * KW)
```

Comment:
@kevinthesun @icemelon9 @comaniac is this okay to autotvm?

Reply:
It's okay in terms of the functionality, but the output message would be weird. Since the AutoTVM progress bar shows throughput instead of latency, users will always see 0 GFLOPS during the tuning process (https://github.com/apache/incubator-tvm/blob/master/python/tvm/autotvm/tuner/callback.py#L159).

Maybe we can still have the FLOPS with N=1 and pop a message saying we are tuning the kernel with N=1 but it can be used by the kernel with any batch size?

Reply:
yeah, I thought about 1 as well. But it actually maybe not 1 as well.

Reply:
I think it's fine since generally AutoTVM can't be used for dynamic shape op. Thus flops info is not useful for dynamic shape op.